web/crux/src/app/node/node.http.controller.ts

```diff
@@ -34,9 +36,11 @@ import NodeService from './node.service'
 import NodeGenerateScriptValidationPipe from './pipes/node.generate-script.pipe'
 import NodeGetScriptValidationPipe from './pipes/node.get-script.pipe'
 
+const PARAM_NODE_ID = 'nodeId'
```

Comment:
Moving this to `node.const.ts` could make it reusable in other controllers as well.

Reply:
Updated.